py/setup.py

```diff
@@ -239,8 +239,6 @@ def run(self):
         libraries=["torchtrt"],
         include_dirs=[
             dir_path + "torch_tensorrt/csrc", dir_path + "torch_tensorrt/include",
-            dir_path + "/../bazel-TRTorch/external/tensorrt/include",
```

Comment:
Might as well keep this since if people dont redownload the repo after the namechange its going to break the build

Reply:
added it back. But it would be nice to clean this up at some point in time.